src/Tools/dotnet-monitor/Extensibility/FolderExtensionRepository.cs

```diff
@@ -28,15 +29,42 @@ public FolderExtensionRepository(IFileProvider fileSystem, ILoggerFactory logger
 
         public override bool TryFindExtension(string extensionName, out IExtension extension)
         {
-            IDirectoryContents extensionDir = _fileSystem.GetDirectoryContents(extensionName);
+            string extensionPath = string.Empty;
+
+            bool isDotnetToolsLocation = _targetFolder == HostBuilderSettings.ExtensionDirectoryPath;
+
+            // Special case -> need to look in particular path for dotnet tool installation
```

Comment:
This logic is for the `dotnet tool install` scenario, so that users don't need to manually place an extension downloaded from Nuget into the right directory. This works in my limited testing (building a nuget package locally, and then calling `dotnet tool install --add-source...` on it), but I think this area may need some more refinement going forward.